features/download_item_list.feature

```diff
@@ -47,3 +41,25 @@ Feature: Managing Item Lists
     And I should see "Download Selected"
     And I should see "Download only files that match a particular regular expression"
     And I should see "Download Matches"
+
+
+  Scenario: Downloading more than 200 documents results in an error
+    Given "researcher@intersect.org.au" has item lists
+      | name  |
+      | Test1 |
+    And the item list "Test1" has 101 remote documents
```

Comment:
I think this is intended as "...has 201 remote documents" rather than "...has 101 remote documents".